keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -761,7 +740,10 @@ Focusing mostly on:
     heartbeats, leader election, etc.)
 -->
 
-No.
+Yes, enabling this feature will result in new API calls.  Specifically:
+
+- Discovery calls via a loopback client: The local apiserver will use a loopback client to discover the resources it serves for each group-version
+- Remote discovery calls to peer apiservers: The event handler for apiserver identity lease informer will make remote discovery calls to each peer apiserver whose lease object is added or updated
```

Comment:
How often is this anticipated to happen? If there are no upgrades going on, are these still expected to happen on a regular interval or only once per apiserver?

Reply:
For the local discovery calls - those should happen only once upon server startup.
For aggregated discovery calls - since those are tied to the apiserver identity lease informer, unless we can identify a way to know when an apiserver was re-started just by looking at a lease-update, these calls will be made everytime the lease was updated. Given that leases are short lived, this could be a concern.

Reply:
> unless we can identify a way to know when an apiserver was re-started just by looking at a lease-update,

This is possible by keeping track of the holderIdentity of the lease object, an update on which means that the apiserver was restarted